src/Kdyby/Translation/Dumper/DoctrineDumper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Kdyby\Translation\Dumper;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\PDOConnection;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Types\Type;
+use Tracy\Debugger;
+
+class DoctrineDumper extends DatabaseDumper
+{
+
+    /** @var Connection */
+    private $conn;
+
+    /** @var AbstractSchemaManager */
+    private $sm;
```

Comment:
Co rozepsat ty dvou a třípísmenkový property? Je to takový ošklivý to takto zkracovat :(